api/SpeechSynthesisEvent.json

```diff
@@ -242,6 +242,58 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "milliseconds": {
+          "__compat": {
+            "description": "<code>elapsedTime</code> in milliseconds",
+            "support": {
+              "chrome": {
+                "version_added": "33"
+              },
+              "chrome_android": {
+                "version_added": "33"
+              },
+              "edge": {
+                "version_added": "14"
+              },
+              "firefox": {
+                "version_added": "49",
+                "version_removed": "95"
+              },
+              "firefox_android": {
+                "version_added": "62",
+                "version_removed": "95"
```

Comment:
Firefox for Linux and Firefox for Android is that `elapsedTime` is second even if old version.  [Bug 1732498](https://bugzilla.mozilla.org/show_bug.cgi?id=1732498) is Windows and macOS only.